server/templates/pages/src/privacy.html

```diff
@@ -1,7 +1,5 @@
 <!doctype html>
-<!--[if lt IE 7]>      <html class="no-js lt-ie10 lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html class="no-js lt-ie10 lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html class="lt-ie10 lt-ie9"> <![endif]-->
+<!--[if lt IE 9]>      <html class="no-js"> <![endif]-->
 <!--[if IE 9]>         <html class="lt-ie10"> <![endif]-->
```

Comment:
I believe R2 and R3 can be combined into a single line:

```html
<!--[if lt IE 10]>      <html class="lt-ie10"> <![endif]-->
```

That way the browse-happy text is displayed, but the rest of the content is not.